### PR TITLE
add gui to repo

### PR DESCRIPTION
**Pull request by magland**
@alejoe91 

Moving spikesortingview to this repo in gui/

The GUI (aka visualization plugin) is deployed via gui/devel/deploy_dev.sh to gs://figurl/spikesortingview-11dev

(using 11dev for now)

And I modified sortingview to create the 11dev links: https://github.com/magland/sortingview/blob/add-gui-to-repo/sortingview/views/View.py

### CODE REVIEW

**Comment by magland** [comment]
@jsoules This is a move to put the GUI code in the same repo as the python code for sortingview.

File: sortingview/views/View.py [hunk 1]
```diff
@@ -77,7 +77,7 @@ def url_dict(self, *, label: str, state: Union[dict, None]=None):
                     for view in all_views if not view.is_layout
                 ]
             }
-            view_url = os.getenv('SORTINGVIEW_VIEW_URL', 'gs://figurl/spikesortingview-10')
+            view_url = os.getenv('SORTINGVIEW_VIEW_URL', 'gs://figurl/spikesortingview-11dev')
```
Comment (alejoe91) [inline sortingview/views/View.py:80]: it would be a good idea to store this in a separate file

**Comment by magland** [comment]
@alejoe91 
> it would be a good idea to store this in a separate file

Done.

**Comment by alejoe91** [comment]
Looks great! Good to merge on my side :)

**Pull request merged**